netci.groovy

```diff
@@ -184,4 +184,46 @@ Utilities.addGithubPRTrigger(prCCJob, 'Code Coverage Windows Debug', '@dotnet-bo
         Utilities.addStandardOptions(newJob)
         Utilities.addXUnitDotNETResults(newJob, 'bin/tests/**/testResults.xml')
     }
+
+// **************************
+// Outerloop and Innerloop against the latest dependencies on Windows. Rolling daily for debug and release
+// **************************
+
+['Debug', 'Release'].each { configuration ->
```

Comment:
Run against latest has not implemented yet for Linux. There is a active issue tracking it.